RevenueCatUI/PaywallView.swift

```diff
@@ -44,34 +47,40 @@ public struct PaywallView: View {
     private var error: NSError?
 
     /// Create a view that loads the `Offerings.current`.
+    /// - Parameter displayCloseButton: Set this to `true` to automatically include a close button.
```

Comment:
Just a question but I noticed we don't have docs for the `fonts` parameter. Should we have it as well?

Reply:
I'll add it now 👍🏻